conf/index.html

```diff
@@ -1,87 +1,116 @@
-<!DOCTYPE html>
-<html>
+<html  xmlns="http://www.w3.org/1999/xhtml"
```

Comment:
could we split this into "*Controller.js", "*Service.js" and leave "*.html" for pure template ?